PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalVaultRequestUnitTest.java

```diff
@@ -52,6 +52,27 @@ public void setsValuesCorrectly() {
         request.setLandingPageType(PayPalRequest.LANDING_PAGE_TYPE_LOGIN);
         request.setShouldOfferCredit(true);
         request.setAppLinkEnabled(true);
+        PayPalBillingInterval billingInterval = PayPalBillingInterval.MONTH;
```

Comment:
An aside: what are people's thoughts on not having to write unit tests for Kotlin data class properties?

I feel that there isn't any real logic that we're testing unless properties have custom setters/getters.

Reply:
I agree - there is no value to testing that properties were set. Like you suggested earlier, we should assert that the JSON is properly formatted from a given Data class set of inputs.

Reply:
I think the only value currently is confirming Java integration doesn't change (i.e. sanity check that the setter pattern is working as expected).

Update to include a JSON formatting unit test too 👍